test/integration/router.specifiedRoutes.test.js

```diff
@@ -193,36 +193,6 @@ describe('router :: ', function() {
 			});
 		});
 
-		describe('with dynamic url paths specified', function() {
```

Comment:
It was asserting things about the internals of express route matching, I believe, specifically the `optional` flags